### PR TITLE
HOCS-3079 Alter application banner when running in a NOTPROD environment

### PR DESCRIPTION
**Pull request by fewtrees-ho**
https://collaboration.homeoffice.gov.uk/jira/browse/HOCS-3079

### CODE REVIEW

File: src/shared/common/components/phase-banner.jsx [hunk 1]
```diff
@@ -3,14 +3,21 @@ import PropTypes from 'prop-types';
 
 export default class PhaseBannerComponent extends Component {
     render() {
-        const { feedback, phase } = this.props;
+        const { feedback, phase, environment } = this.props;
+
+        var envDisplay;
```
Comment (robertdeniszczyc2) [inline src/shared/common/components/phase-banner.jsx:8]:
```suggestion
        let envDisplay;
```

File: src/shared/common/components/__tests__/phase-banner.spec.js [hunk 2]
```diff
@@ -17,4 +17,9 @@ describe('Phase Banner component', () => {
             render(<PhaseBanner feedback="http://some.domain" />)
         ).toMatchSnapshot();
     });
+    it('should display the correct test notice when NOTPROD', () => {
+        expect(
+            render(<PhaseBanner environment="NOTPROD" />)
+        ).toMatchSnapshot();
+    });
```
Comment (robertdeniszczyc2) [inline src/shared/common/components/__tests__/phase-banner.spec.js:24]: Also add a test for displaying the correct message when prod

**Review by robertdeniszczyc2 — CHANGES_REQUESTED** [review]
Added a couple minor comments

File: src/shared/common/components/phase-banner.jsx (hunk 1 above)
Comment (fewtrees-ho) [reply src/shared/common/components/phase-banner.jsx]: changed to let in the new code

File: src/shared/common/components/__tests__/phase-banner.spec.js (hunk 2 above)
Comment (fewtrees-ho) [reply src/shared/common/components/__tests__/phase-banner.spec.js:29]: Added explicit test for this - there was a default test but explicit is better

**Review by robertdeniszczyc2 — APPROVED** [review]
Looks good to me

**Pull request merged**